### DISCUSSION TITLE
Help wanted

### DISCUSSION THREAD

**Discussion by eduard-grigoryan-ext-bark** [shlinkio/shlink#2260] [Help wanted] [answered]
### Shlink version

2.10

### PHP version

8.1

### How do you serve Shlink

Docker image

### Database engine

MySQL

### Database version

8

### Summary

Does anybody now in terms security and vulnerabilities, is there difference between 2.x versions and 4.x versions in shlink itself?

**Comment by acelaya** [comment, marked as answer]
Absolutely. Get updated ASAP
